global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": [ "src", "test", "samples" ]
+    "projects": [ "src", "test", "samples" ],
+    "sdk": {
+        "version": "1.0.0-preview1-002702"

```

Comment:
That corresponds to the version that you get of the tooling etc when you install the bits from https://www.microsoft.com/net/core

> dotnet --info
> .NET Command Line Tools (1.0.0-preview1-002702)